pkg/cmd/server/crypto/crypto.go

```diff
@@ -455,7 +453,20 @@ func NewKeyPair() (crypto.PublicKey, crypto.PrivateKey, error) {
 }
 
 // Can be used for CA or intermediate signing certs
-func newSigningCertificateTemplate(subject pkix.Name) (*x509.Certificate, error) {
+func newSigningCertificateTemplate(subject pkix.Name, expireDays int) (*x509.Certificate, error) {
+
+	var caLifetimeInDays = defaultCACertificateLifetimeInDays
+	if expireDays > 0 {
+		caLifetimeInDays = expireDays
+	}
+
+	if caLifetimeInDays >= maxRecommendedLifetimeInDays {
```

Comment:
Yeah, we've discussed 10 as possible longer period. But we've mentioned we'll warn at anything longer than 2 years, or even a year.